### DISCUSSION TITLE
GenMachine Ren5000 - Ryzen 5700U

### DISCUSSION THREAD

**Discussion by SalmonPete** [ChefKissInc/NootedRed#239] [Show and tell]
Just confirmation that Sonoma 14.4.1 works great with my 5700U and the latest NootedRed.kext. Amazing work thank you.
![Screenshot 2024-03-28 at 9 59 22 AM](https://github.com/ChefKissInc/NootedRed/assets/143473778/866bdc0b-3c39-451e-9e39-831e837a288f)

**Comment by SalmonPete** [comment]
Successfully installed macOS Sequoia Beta 2 on my GenMachine, stability, imo, is better than Sonoma.

![Screenshot 2024-06-28 at 11 55 59 PM](https://github.com/ChefKissInc/NootedRed/assets/143473778/95c4bdcf-3627-446a-b246-20874dee58b3)

**Reply by BlueNecko** [reply]
Have you tried enabling hidpi, try and tell me if your system lags

**Reply by SalmonPete** [reply]
Forgive me if I've misunderstood your request and posted nonsense. I downloaded SwitchRes-X and BetterDisplay, though I couldn't find any hidpi options in BetterDisplay, SwtchRes-X did allow me turn it on. I switched to 2560x1440 @240MHz and haven't noticed anything lagging, yet. This is on the a 4700G system though I will test the 5700U and report back.

![Screenshot 2024-06-29 at 9 42 55 PM](https://github.com/ChefKissInc/NootedRed/assets/143473778/ba9ca90b-4166-4ffe-9adc-fa6200d81639)

**Reply by BlueNecko** [reply]
You can try following the guide on better display's wiki. It tell how to enable it https://github.com/waydabber/BetterDisplay/wiki . My system has a r5 5600H and it lags when enabling hidpi, and I am not sure if it's not able to handle that resolution or it's just a driver glitch. Do tell me what you find.

**Reply by SalmonPete** [reply]
I'll have more time tomorrow to take a look at BetterDisplay. In the meantime here's a screenshot of macOS Sequoia Beta 2 in 2560x1440 (Scaled) at 240Hz, this is on the 5700U GenMachine and I'm not getting system lag.

![Screenshot 2024-06-29 at 10 11 37 PM](https://github.com/ChefKissInc/NootedRed/assets/143473778/bbdb1d4e-b064-447f-80ea-3d15a76868a1)

**Comment by SalmonPete** [comment]
macOS Sequoia Public Beta 2 successfully installed and updated to the latest NootedRed.kext, all good.

![Screenshot 2024-07-31 at 1 16 45 AM](https://github.com/user-attachments/assets/0b556bae-f25a-475d-b960-6d55216362cb)

**Reply by pdx-rico** [reply]
Heya!
Can u provide your EFI folder? I have similar miniPC
